Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -60,29 +68,37 @@ class HistoryPanel: UIViewController, HomePanel {
         tableViewController.historyPanel = self
 
         self.addChildViewController(tableViewController)
-        self.view.addSubview(tableViewController.view)
-        self.view.addSubview(syncedTabsButton)
-        self.view.addSubview(recentlyClosedTabsButton)
 
-        recentlyClosedTabsButton.snp_makeConstraints { make in
-            make.top.leading.trailing.equalTo(self.view)
-            make.height.equalTo(HistoryPanelUX.SyncedTabsCellHeight)
-            make.bottom.equalTo(syncedTabsButton.snp_top)
-        }
+        setUpHistoryPanelViews()
+        
+        self.tableViewController.didMoveToParentViewController(self)
+    }
 
-        syncedTabsButton.snp_makeConstraints { make in
-            make.leading.trailing.equalTo(self.view)
-            make.top.equalTo(recentlyClosedTabsButton.snp_bottom)
-            make.height.equalTo(HistoryPanelUX.SyncedTabsCellHeight)
-            make.bottom.equalTo(tableViewController.view.snp_top)
-        }
+    func setUpHistoryPanelViews() -> Success {
```

Comment:
Nit: If you don't need to chain anything after this method, you can just return `Void` and replace the `bindQueue` with `uponQueue`.